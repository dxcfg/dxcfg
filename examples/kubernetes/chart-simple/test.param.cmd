npm ci
jk generate --stdout ./index.js -p values.name=demo -p values.image.tag=v2
