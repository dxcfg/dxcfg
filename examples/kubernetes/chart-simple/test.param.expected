apiVersion: apps/v1
kind: Deployment
metadata:
  name: demo-dep
spec:
  template:
    labels:
      app: hello
    spec:
      containers:
        hello:
          image: weaveworks/helloworld:v2
---
apiVersion: v1
kind: Service
metadata:
  labels:
    app: hello
  name: demo-svc
spec:
  selector:
    app: hello
